lerna.json

```
@@ -9,5 +9,8 @@
     "common/predictive-text",
     "web"
   ],
+  "hoist": [
+    "typescript"
+  ],
```

This fixes the failure with `The batch file cannot be found.`. Turns out Lerna is deleting itself (which just happens to work on macOS because we don't do anything else with Lerna after bootstrap).

I cannot find documentation on the `hoist` property of lerna.json though. Don't know if this is recommended or naughty. However, this does also restore the 65 bazillion lines of package.json!

```suggestion
  "hoist": [
    "lerna",
    "typescript"
  ],
```

The build is still failing with the other error I noted though!

---

Nor could I, but I like it.  Looks like we manage the hoisted versions through the base-level package.json; I've just pushed up a commit to this effect.

Updates are then done by updating the package in the base-level package.json, as if it were a normal npm package.json, then lerna-bootstrapping again.